### PR TITLE
Fix NYC to check all sources.

### PR DESCRIPTION
**Pull request by Mossop**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,5 +66,11 @@
   },
   "engines": {
     "node": ">=5.8.0"
+  },
+  "nyc": {
+    "all": true,
+    "require": "babel-register",
+    "include": ["app/**"],
+    "extension": [".jsx"]
```
Comment (victorporof) [inline package.json]: Strange that this array doesn't contain '.js' in it.

File: .babelrc [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {
-  "presets": [ "es2015-node5", "stage-2", "react" ]
+  "presets": [ "es2015-node5", "stage-2", "react" ],
+  "sourceMaps": "inline"
```
Comment (victorporof) [inline .babelrc]: Kinda' sad that we have to inline sourcemaps.

Comment (Mossop) [inline .babelrc]: It looks like this is only applying to when babel-register is used, we can certainly override that for the actual code we run in task-build.js

File: package.json (hunk 1 above)
Comment (Mossop) [inline package.json]: Yeah, seems to work though.

**Comment by ncalexan** [comment]
Between this and #209, this looks good.

**Pull request merged**